kroxylicious-api/src/main/java/io/kroxylicious/proxy/service/Contributor.java

```diff
@@ -7,46 +7,42 @@
 
 import io.kroxylicious.proxy.config.BaseConfig;
 
+import edu.umd.cs.findbugs.annotations.NonNull;
+
 /**
  * Support loading an Instance of a service, optionally providing it with configuration obtained
  * from the Kroxylicious configuration file.
  *
  * @param <T> the service type
+ * @param <S> the context type
```

Comment:
Please can we make these `<S>` for the `S`ervice type and `<C>` for the `C`ontext type?